src/routes/styleguide/components/ColorRenderer.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
```

Comment:
Secondly, the `src/routes` founder is meaningful in SvelteKit as it adds it to the router. This means that a user _could_ navigate to `/styleguide/components/ColorRenderer` as well as `/styleguide/components`, which will error.

Shared components used throughout the codebase can go into `$lib/components` (this is a SvelteKit convention).

If you think you're only going to use a component with a given route, you can prefix it with an underscore (e.g. `_ColorRenderer.svelte`.

Also: We're currently going with kebab case (e.g. `color-renderer.svelte`. We can argue about whether or not that is the right choice for Svelte components, but for now we might as well stay consistent.